platform/machine/qemu/cluster.go

```diff
@@ -256,6 +258,22 @@ func (qc *Cluster) NewMachineWithOptions(userdata *conf.UserData, options Machin
 		return nil, err
 	}
 
+	// wait in the background for qemu to die so we can tell if it was killed (as it should be) or ran in to
+	// some other problem and died.
+	go func() {
```

Comment:
It seems like this should be synchronous in `qm.Destroy()`.  It'll probably still need to use `plog`, since no one checks the return value from `Destroy`, but at least it avoids the goroutine.